src/slurm/Makefile.am

```diff
@@ -0,0 +1,23 @@
+
+plugindir = $(libdir)/slurm
```

Comment:
I think the SLURM intent is to have third-party libraries install into their plugin directory.  This is how other plugins appear to do it, e.g., 

https://github.com/grondo/slurm-spank-plugins/blob/master/Makefile

With the plugin installed in the plugin directory, then the admin only has to specify the plugin name:
```
required singularity.so
```
as opposed to the full path.

Looking at the SLURM RPM, _only_ plugins are put into this directory: none of the core libraries or executables live here.